.codex/skills/gwt-fix-issue/scripts/inspect_issue.py

```
@@ -0,0 +1,786 @@
+    def __init__(self, returncode: int, stdout: str, stderr: str):
+        self.returncode = returncode
+        self.stdout = stdout
+        self.stderr = stderr
+
+
+# =============================================================================
+# Git and GH utilities
+# =============================================================================
+
+def run_gh_command(args: Sequence[str], cwd: Path) -> GhResult:
+    process = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)